notebooks/ch14/bias_modeling.md

```diff
@@ -204,15 +228,25 @@ Cross-validation provides a more accurate method of estimating our model error u
 
 The bias-variance tradeoff allows us to more precisely describe the modeling phenomena that we have seen thus far.
 
-Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance.
+Underfitting is typically caused by too much bias; overfitting is typically caused by too much model variance.
 
-Collecting more data reduces variance. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
+Collecting more data reduces variance. In the extreme case that we can collect all the data for something, we would be able to create a model with zero variance- meaning that it predicts every single point correctly. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
```

Comment:
for example, model variance of linear regression goes down as 1/n . Thus, doubling the dataset halves the model variance, and collecting many data points will cause the variance to approach 0.